### PR TITLE
Project: Building with Backgrounds and Gradients

### PR DESCRIPTION
**Pull request by krys2fa**
Milestones:
 - built a webpage using:
          -- images as a background 
          -- and adding gradients to elements. 

The goal was to make a clone of old an old version of the Apple website webpage.

### CODE REVIEW

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Comment by mikenath223** [comment]
**Pull request currently under review by Michgolden Ukeje (TSE).**
Kindly exercise patience 👋

File: index.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  				<li>
+  					<a href="#">
+  						<img src="./images/mac-birthday.jpg" alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+  					</a>
+  				</li>
+
+  				<li>
+  					<a href="#">
+  						<img src="./images/supplier-responsibility.jpg" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+  					</a>
```
Comment (mikenath223) [inline index.html]: [ ] **Required** -These shouldn't be images also kindly ensure you make these backgrounds also.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+  	<div class="main-container flex-v">
+
+  		<div class="header">
+  			<div class="nav-bar">
+	  			<ul>
+	  				<li><a href="#">
+	  						<img class="logo-img" src="./images/apple-logo.png" alt="">
+	  					</a>
+	  				</li>
```
Comment (mikenath223) [inline index.html:21]: - [ ] **Required** - These shouldn't be images as this is a project focused on backgrounds and gradients kindly ensure you make these backgrounds instead.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
HI @krys2fa  👋

Good job on this.👌

However, there are a few areas that need to be adjusted:

- [ ] Required - Kindly enable stickler on your repo and ensure any existing linting errors are fixed.

- [ ] Required - Also please fix the issue in your navbar as shown below. There is a search input field expected to be on the right side of the navbar. And a search icon in it.
**THIS IS YOURS👇**
ice_screenshot_20200408-190206

**THIS IS THE WAY IT SHOULD BE👇**
ice_screenshot_20200408-191024

**ALSO, THERE ARE ISSUES ON PORTIONS OF YOUR CODE. They are described below. Kindly scroll down and view each highlighted code area, and fix the issues described there.**

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

**KINDLY SCROLL DOWN AND VIEW THE ISSUES IN YOUR CODE.👇👇**

File: index.html (hunk 2 above)
Comment (krys2fa) [reply index.html:21]: Thanks @mikenath223. The images have been removed and used as backgrounds as instructed.

**Comment by krys2fa** [comment]
Hello @mikenath223,

All issues have been worked on.  Thank you.

Regards,

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️ 🚥 

Hi @krys2fa, you have done a great job fixing the recommended changes required by the previous TSE 👍 However, in fixing the changes, we developed some bugs we should fix. Kindly check comments below:
- Kindly check the image below and fix the horizontal scroll and the navbar 🙂 
- There should be no margin between the main content and the 4 column section as in the image below
fix1

[OPTIONAL]
Going forward, let us improved our commit messages, for example, use ```add``` instead of ```added``` and so on
Kindly make these changes and submit another code review 😄 
Happy coding_ 👨‍💻 
Daniel Larbi Addo on slack

**Comment by krys2fa** [comment]
Hello @addod19,

Thanks for the review. I have implemented the corrections.

Best regards,

**Comment by christianotieno** [comment]
## Pull Request under review by Christian Otieno (TSE)

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @krys2fa  :wave:
-
You have done a splendid job! :tada: We are almost there. Keep it up.
We do have some few changes to carry out before we proceed (see review comment).
##
Good job implementing the required changes.
As with the past reviews, the header appears not have been fixed properly. Kindly fix that and add links to 4 elements as well. As with the width of the page lets reduce it match the original version. Thank you. 
- From this: 👇 
apple_clone
- To match this: 👇 
apple

Please make the suggested changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. :eyes:
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by krys2fa** [comment]
Hello Christian,

Thanks for the review. I have worked on the requested changes.

Kind regards,

**Comment by sumancrest0001** [comment]
### STATUS: CHANGES REQUIRED

**Hello @krys2fa** 
Great work to fix all the issues addressed by the previous TSE :clap: :clap:  and you are an inch closer to get approval from our side. We have to fix a couple of minor issues and I am pretty much sure that it won't take much time to fix them.

[REQUIRED] We have to use **float and flex or grid** to align elements. Currently, you are using flex only. Please use float also to tick the requirement. [Requirement from the pathright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/) (project-specific section)

[Optional] Please add underline effect when hovered over :point_down: :point_down:  line.
Screenshot from 2020-04-09 17-05-47

[Optional] Please use semantic HTML tags like **main and nav** instead of just div tag.

**After making the change. Request for another review. 
:+1:  :+1: 
Happy coding**

**Comment by krys2fa** [comment]
Hello @sumancrest0001, 
Thanks for the review. Please note that the requested changes have been implemented.
Regards,

**Comment by Dandush03** [comment]
### :heavy_check_mark: Project Approve :exploding_head: :checkered_flag:
# Hey @krys2fa   :wave: 
### Great Job so Far :grey_exclamation: :grey_exclamation: :muscle: :smiley: 

### Following Previous Reviews and Seeing you Have Made Required Changes. 
### The project Looks Great.:grey_exclamation::wink:.

### Good Luck in your Next Assignment
### Happy coding:computer: 
#### Daniel Laloush

**Comment by krys2fa** [comment]
Thank you @Dandush03.

**Pull request merged**